server/app/services/export/CsvExporter.java

```diff
@@ -62,7 +66,20 @@ public void exportRecord(
     ImmutableMap<Path, ApplicantQuestion> questionMap =
         roApplicantProgramService
             .getAllQuestions()
-            .collect(ImmutableMap.toImmutableMap(aq -> aq.getContextualizedPath(), aq -> aq));
+            .collect(
+                ImmutableMap.toImmutableMap(
+                    aq -> aq.getContextualizedPath(),
+                    Function.identity(),
+                    // TODO(#9212): There should never be duplicate entries because question paths
+                    // should be unique, but due to #9212 there sometimes are. They point at the
+                    // same location in the applicant data so it doesn't matter which one we keep.
+                    (existing, replacement) -> {
+                      LOGGER.warn(
+                          "Duplicate questions with path '%s', which may be resulting in data"
```

Comment:
```suggestion
                          "Duplicate questions with path '{}', which may be resulting in data"
```

Logback uses `{}` for placeholders.